cloudferrylib/os/network/neutron.py

```diff
@@ -279,7 +279,6 @@ def convert_subnets(snet, cloud):
         }
 
         res_hash = network_res.get_resource_hash(result,
-                                                 'name',
```

Comment:
I think subnet name meaningless when we already have the same subnet on dst except name.
In this way if someone manually creates subnet on dst with needed `cidr`, gateway, ect, we will use it.